appinventor/appengine/src/com/google/appinventor/client/boxes/SourceStructureBox.java

```diff
@@ -52,4 +54,14 @@ private SourceStructureBox() {
   public SourceStructureExplorer getSourceStructureExplorer() {
     return sourceStructureExplorer;
   }
+
+  public SourceStructureExplorer show(YaFormEditor form) {
```

Comment:
It doesn't seem like the return value is used anywhere. Maybe this should just return void?